_posts/2013-01-14-alt-text.md

```diff
@@ -1,21 +1,29 @@
 
-- Use the `alt` attribute for any image that is used as content. `alt` text describes the function of an image and is rarely a detailed description of the image itself (unless, for example, the page is actually about a specific picture).
-- Make sure the description of the image is useful. For example, if the image is your logo your `alt` should be your company name and not "logo".
-- Don't begin `alt` text with "photo of.." or "picture of.."; assistive technologies already do this. Redundancy and repetition are almost as bad as not enough information.
-- [Don't put line breaks in `alt` text.](https://developer.paciellogroup.com/blog/2015/09/short-note-on-coding-alt-text/) It causes suboptimal effects when read by screen readers.
-- Using an image file several times in a website doesn't mean that the `alt` attribute has to be the same in each `img` tag, because `alt` attributes change with context. For example, as seen above, a logo that is a link to your home page should have `alt="home page"` and the same logo, used somewhere else as decoration, should have `alt=""`.
-- Use an empty `alt` attribute for any image that is decorative or not necessary for understanding the content of the page (`alt=""` or just `alt` - e.g. `<img src="prettyswirl.jpg" alt>`). If, for example, the information in a graph is explained in text below it, there's no need to duplicate that in `alt` text.
-- Images that are links should describe where they go. For example, a logo that is a link to your home page should have `alt="home page"`. But if the image is only one part of the contents of a link and the other contents (e.g. plain text) describes the link then use an empty `alt` attribute for the image - if you were to specify 'alt' text as well this would be redundant/repetitive.
+## A few tips on how and when to use the `alt` attribute:
+- Use the `alt` attribute for images that are used as more than decoration. The alternative text describes the primary purpose of an image and is rarely a detailed description of the image itself (unless, for example, the page is actually about a specific image).
+- Make sure the description of the image is useful. For example, if the image is of a company's logo, the `alt` should (often) be the company's name. The word "logo" is not necessary or useful as part of the alternative text.
+- Don't begin alternative text with "photo of..." or "picture of..."; assistive technologies already indicate the role of the element as an "image" or "graphic". Unnecessary redundancy makes for poor user experiences.
+- [Don't put line breaks in `alt` text](https://developer.paciellogroup.com/blog/2015/09/short-note-on-coding-alt-text/). It causes suboptimal effects when announced by screen readers.
```

Comment:
I'd rephrase this to use something other than suboptimal. It's kinda a jargon-adjacent term in an situation where the reader may not already know what alt text is supposed to sound like. Maybe describe what the effect would be, for those who don't read the link? That or just say something like, "It can break how a screen reader reads the image's description."